packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsPendingRow/TransactionsPendingRow.tsx

```diff
@@ -141,6 +141,7 @@ const TransactionsCryptoRow: React.FC<TProps> = ({ transaction }) => {
                     name='Time'
                     value={moment
                         .unix(transaction.submit_date)
+                        .utc()
```

Comment:
so I'm assuming, that transaction.submit_date is unix timestamp,
and we always want to display it in GMT no matter the end user timezone?

Reply:
yep correct, currently without this fix the timestamp shown would be the local timezone